src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.46.0"
+__version__ = "0.46.1"
```

Comment:
Bumped only patch version because it is a small change

Reply:
Since its a new function - ideally it should be a minor version bump. Approving to unblock anyway

Reply:
I also thought the same. Since I planned to update only prompt service, made it as a patch.

Let me update minor version itself.